app/src/main/res/values-it/strings.xml

```diff
@@ -308,33 +308,34 @@ ed anonima"</string>
 
     <!-- Hinweis Titel -->
     <string name="hinweis_title">"Avviso importante"</string>
 
     <!-- Hinweis Text -->
-    <string name="hinweis_text">"Una persona che hai incontrato è risultata positiva alla COVID-19."</string>
-    <string name="button_report_illness">"Informa della malattia"</string>
+    <string name="hinweis_text">"Una persona con cui sei stato co-presente è risultata positiva alla COVID-19."</string>
+    <string name="button_report_illness">"Informa della potenziale malattia"</string>
     <string name="meldungen_meldung_title">"Avviso importante"</string>
-    <string name="meldungen_meldung_text">"Una persona che hai incontrato è risultata positiva alla COVID-19. Informarti qui per le raccomandazioni ed il comportamento da adottare."</string>
+    <string name="meldungen_meldung_text">"Una persona con cui sei stato in co-presenza è risultata positiva alla COVID-19. Informarti qui per le raccomandazioni ed il comportamento da adottare."</string>
     <string name="meldungen_meldung_more_button">"Scopri di più"</string>
     <string name="tracing_active_title">"Tracciatura attiva"</string>
-    <string name="tracing_active_text">"Gli incontri vengono salvati in modo anonimo sul tuo dispositivo."</string>
+    <string name="tracing_active_text">"Le co-presenze vengono conteggiate in modo anonimo sul tuo dispositivo."</string>
     <string name="tracing_error_title">"Tracciatura non attiva"</string>
-    <string name="tracing_error_text">"Attiva la tracciatura e il Bluetooth in modo che gli incontri possano essere registrati in modo anonimo."</string>
+    <string name="tracing_error_text">"Attiva la tracciatura e il Bluetooth in modo che le co-presenze possano essere registrate in modo anonimo."</string>
 
     <!-- Meldungen: zusätzliche Info Titel -->
     <string name="meldungen_additional_info_title">"Cosa significa?"</string>
 
     <!-- Meldungen: Info Text 1 -->
-    <string name="meldungen_additional_info_text1">"Per interrompere la diffusione di un virus, è necessario interrompere la catena di infezione. Puoi farlo isolando le persone contagiose. C'è però un problema con il virus COVID-19: potresti essere contagioso prima di sentirti male! Ciò consente al virus di diffondersi, anche se le persone con sintomi si isolano questo perché siamo un passo indietro."</string>
+    <string name="meldungen_additional_info_text1">"Per interrompere la diffusione di un virus, è necessario interrompere la catena d'infezione. Puoi farlo isolando le persone contagiose. C'è un problema con il virus della COVID-19:
+potresti essere contagioso prima di avvertirne i sintomi o in caso fossi asintomatico! Ciò consente al virus di diffondersi se non ci confiniamo, in tempo più breve possibile. Cosi facendo, il nostro sistema sanitario risulterebbe essere sempre un passo indietro per la lotta contro la COVID-19."</string>
```

Comment:
se non ci confiniamo, in tempo più breve possibile
-->
se non ci confiniamo nel più breve tempo possibile

Reply:
corretto.